scripts/record_episodes.py

```diff
@@ -395,4 +410,9 @@ def debug():
             ' a joystick controller or some other manual method.'
         ),
     )
+    parser.add_argument(
+        '--gravity_compensation',
+        action='store_true',
+        help='If set, gravity compensation will be enabled for the leader robots.',
```

Comment:
If set, gravity compensation will be enabled for the leader robots when teleop starts.